nautobot/core/apps/__init__.py

```diff
@@ -96,14 +96,17 @@ def register_menu_items(tab_list):
     """
     for nav_tab in tab_list:
         if isinstance(nav_tab, NavMenuTab):
-            create_or_check_entry(registry["nav_menu"]["tabs"], nav_tab, nav_tab.name, f"{nav_tab.name}")
+            # Handle Apps that haven't been updated yet
+            if nav_tab.name == "Plugins":
+                nav_tab.name = "Apps"
+            create_or_check_entry(registry["nav_menu"]["tabs"], nav_tab, nav_tab.name, nav_tab.name)
```

Comment:
Not in scope for your issue, but if you're feeling bored, a docstring for `create_or_check_entry` would be really nice